firestore/cloud-client/snippets.py

```diff
@@ -883,3 +883,67 @@ def update_document_increment(db):
 
     washington_ref.update("population", firestore.Increment(50))
     # [END fs_update_document_increment]
+
+
+# [START fs_counter_classes]
+import random
+from google.cloud import firestore
+
```

Comment:
Can you add a newline between the imports? Generally, it's best to split the imports up into separate blocks of standard library, third-party and package-level as per https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#style--linting